README.md

```diff
@@ -1462,6 +1462,50 @@ jobs:
           publish-summary: false
 ```
 
+### Automatic PR number & URL detection
+
+When recording runs to Cypress Cloud, the PR number and URL can be automatically detected if you pass `GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}` 
+via the workflow `env`. When set, this value enables the Action to perform additional logic that grabs the related PR number and URL (if they
+exist) and sets them in the environment variables `CYPRESS_PULL_REQUEST_ID` and `CYPRESS_PULL_REQUEST_URL`, respectively.
+* See Cypress' documentation on [CI Build Information](https://docs.cypress.io/guides/continuous-integration/introduction#CI-Build-Information)
+
+Example workflow using the variables:
+```yml
+name: Example echo PR number and URL
+on: push
+jobs:
+  cypress-run:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Cypress run
+        uses: cypress-io/github-action@v5
```

Comment:
```suggestion
        uses: cypress-io/github-action@v5
        with:
          record: true
```

You have to tell the action to record 🙂 !

Reply:
🤦 oops lol.

Thanks!